inference-engine/tests/functional/shared_test_classes/src/subgraph/matmul_act_eltw.cpp

```diff
@@ -0,0 +1,47 @@
+// Copyright (C) 2021 Intel Corporation
```

Comment:
Are you testing different eltwise operations? I see only add in the test.
Let's rename the test to mamul_tanh_add

Reply:
and what activation type was used in the legacy test?

Reply:
> and what activation type was used in the legacy test?

Also Tanh, the graph attached in the description